latest/index.bs

```diff
@@ -242,9 +250,36 @@ keys as specified below for discovering certain types of data, especially images
 
 If part of [[#multiscale-md]], the length of "axes" MUST be equal to the number of dimensions of the arrays that contain the image data.
 
+"bioformats2raw.layout" (transitional) {#bf2raw}
+------------------------------------------------
+
+[=Transitional=] "bioformats2raw.layout" metadata identifies a group which implicitly describes a series of images.
+The need for the collection stems from the common "multi-image file" scenario in microscopy. Parsers like Bio-Formats
+define a strict, stable ordering of the images in a single container that can be used to refer to them by other tools.
+
+In order to capture that information within an OME-NGFF dataset, `bioformats2raw` internally introduced a wrapping layer.
+The bioformats2raw layout can be found in NGFF versions v0.2, v0.3, and v0.4. An upcoming NGFF specification will replace
+this layout with explicit metadata.
+
+<pre class=include-code>
+path: examples/bf2raw/bioformats2raw_layout.json
+highlight: json
+</pre>
+
+Conforming groups:
+- MUST have the value "3" for the "bioformats2raw.layout" key in their `.zattrs` metadata at the top of the hierarchy:
+- MUST store separate "multiscales" images in consecutively numbered groups starting from 0 (i.e. "0/", "1/", "2/", "3/", ...);
```

Comment:
At least for backwards compatibility, the primary exception to this rule would be HCS OME-NGFF dataset with metadata stored as `OME/METADATA.ome.xml`. In that case the top-level group (`plate`) MUST include the `bioformats2raw.layout` key but the group hierarchy should remain compliant with the HCS specification described below (i.e. 2 levels of hierarchies for wells with alphanumeric characters)

Reply:
Good catch. I was a bit unsure how to work this in without having ever more complicated if/then logic. See what you think about the version I just pushed which basically says the "plate" takes precedence.